src/components/Tooltip/Tooltip.tsx

```diff
@@ -43,7 +42,7 @@ export const Tooltip: FC<TooltipProps> = ({
       style={style}
       theme={theme}
       trigger={trigger}
-      className={className}
+      class={className}
       {...props}
     >
       {children}
```

Comment:
```suggestion
      {local.children}
```

Use local for reactivity.